app/views/admin/users/_reports.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row my-3">
+  <div class="col-12">
+    <h2 class="d-inline">Recent Reports</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="col-12 collapse" id="reactions-row">
+    <div class="list-group-flush">
+    <% unless @related_reports.empty? %>
```

Comment:
Would it be worth it for me to use `counter_culture` here or should I stick to `.empty?` to follow the convention of the other partials like `_notes.html.erb`?